drivers/include/pca9633.h

```diff
@@ -71,7 +71,7 @@ typedef struct {
 typedef enum {
     PCA9633_OK              = 0,    /**< Success */
     PCA9633_ERROR_I2C       = 1,    /**< I2C communication error */
-} pca9685_error_t;
+} pca9633_error_t;
```

Comment:
IMHO, this should simply be changed to an anonymous enum:

```c
enum {
    PCA9633_OK              = 0,    /**< Success */
    PCA9633_ERROR_I2C       = 1,    /**< I2C communication error */
};
```

It's done this way for return codes in other drivers.